pkg/kubelet/cm/cpumanager/policy_static.go

```diff
@@ -52,6 +56,15 @@ type SMTAlignmentError struct {
 	CausedByPhysicalCPUs  bool
 }
 
+// PodLevelResourcesError represents an error due to pod-level resources not being supported.
+type CPUManagerPodLevelResourcesError struct{}
```

Comment:
Can we use a generic error type for this, rather than a separate CPU & memory manager one?

Maybe something like

```go
type UnsupportedTopologyPolicyError struct{
	Policy string // one of `cpu:static`, `memory:static` ?
	Reason string // might not be needed?
	Message string
}
```